apps/mobile/src/shared/constants.ts

```diff
@@ -5,13 +5,12 @@ export const HEADER_HEIGHT = 64;
 export const GITHUB_ORG = 'leather-io';
 export const GITHUB_REPO = 'extension'; // TODO: we need to create 'mono' config;
 
-export const LEATHER_GUIDES_MOBILE_ANALYTICS = `${LEATHER_GUIDES_URL}/mobile-analytics`;
-export const LEATHER_GUIDES_MOBILE_ACCOUNT_IDENTIFIER = `${LEATHER_GUIDES_URL}/mobile-account-identifier`;
-export const LEATHER_GUIDES_MOBILE_BITCOIN_UNIT = `${LEATHER_GUIDES_URL}/mobile-bitcoin-unit`;
-// TODO: Add this to Leather Guides on Framer
-export const LEATHER_GUIDES_MOBILE_APP_AUTHENTICATION = `${LEATHER_GUIDES_URL}/mobile-app-authentication`;
-export const LEATHER_GUIDES_MOBILE_REMOVE_WALLET = `${LEATHER_GUIDES_URL}/mobile-remove-wallet`;
-
-// TODO: On Framer but no direct link from App
-export const LEATHER_GUIDES_MOBILE_SWITCH_NETWORKS = `${LEATHER_GUIDES_URL}/mobile-switch-networks`;
-export const LEATHER_GUIDES_MOBILE_ADD_WALLET = `${LEATHER_GUIDES_URL}/mobile-add-wallet`;
+export enum GuideLinks {
```

Comment:
Any reason for enum vs plain object?

Reply:
None, other than consistency with other similar usages. I much prefer objects actually.